modules/ROOT/pages/backup-restore/copy-database.adoc

```diff
@@ -210,6 +210,18 @@ The block format is the default format for all newly-created databases as long a
 For more information on the block format, see xref:database-internals/store-formats.adoc[Store formats].
 ====
 
+[NOTE]
+====
+In Neo4j 2025.01, a new dual-named option `--from-pagecache, --max-off-heap-memory=<size>` replaces the `--from-pagecache=<size>` option, which was used to specify how much cache to allocate when reading the source.
+
+The new option determines the maximum amount of off-heap memory available during the copy process for reading and writing.
+This change helps you to control off-heap memory rather than just allocate cache.
+
+To configure the off-heap memory value, you can use either `--from-pagecache=<size>` or `--max-off-heap-memory=<size>`.
```

Comment:
This sentence claims that both are available, but we don't have descriptions and values for both, and we don't explain when to use one or another or both together.

Reply:
Yes, that's correct, they are both available. We have one description and one default value for both.

if I understood the idea correctly, you still can use the old `--from-pagecache=<size>`, but now it will control the off-heap memory value, not the cache allocation and its default value now is `90%` not `8m`.

You cannot use them together. You have to use either  the old name`--from-pagecache=<size>` or the new name `--max-off-heap-memory=<size>`.

Reply:
> Yes, that's correct, they are both available. We have one description and one default value for both. In other words, they share one description and one default value.
> 
> if I understood the idea correctly, you still can use the old `--from-pagecache=<size>`, but now it will control the off-heap memory value, not the cache allocation and its default value now is `90%` not `8m`.
> 
> You cannot use them together. You have to use either the old name`--from-pagecache=<size>` or the new name `--max-off-heap-memory=<size>`.

If that's true, then we need. to change this sentence to say it: